src/components/WorkerTypesTable/index.jsx

```diff
@@ -154,6 +147,13 @@ export default class WorkerTypesTable extends Component {
           return sort(firstElement, secondElement);
         }),
       };
+    },
+    {
+      serializer: ([workerTypesConnection, sortBy, sortDirection]) => {
```

Comment:
`awsProvisionerWorkerTypeSummaries` is missing after `workerTypesConnection`.

Reply:
> `awsProvisionerWorkerTypeSummaries` is missing after `workerTypesConnection`.

It was giving me error that `awsProvisionerWorkerTypeSummaries` is defined but not used. It was not compiling code successfully so I removed it. Do you want me to add ignoring the error?

Reply:
![screenshot from 2018-09-27 19-46-09](https://user-images.githubusercontent.com/13648547/46154222-19469200-c28e-11e8-8c9c-e98651850ca9.png)


Check this error. I can add it back If you want.

Reply:
You can supress this warning by doing:

```jsx
serializer: ([
  workerTypesConnection,
  // eslint-disable-next-line no-unused-vars
  awsProvisionerWorkerTypeSummaries,
  sortBy,
  sortDirection,
])
```